sbi/inference/base.py

```diff
@@ -152,6 +152,69 @@ def __init__(
             median_observation_distances=[], epochs=[], best_validation_log_probs=[],
         )
 
+    def provide_presimulated(
+        self, theta: Tensor, x: Tensor, from_round: int = 0
+    ) -> None:
+        r"""
+        Provide external $\theta$ and $x$ to be used for training later on.
+
+        Args:
+            theta: Parameter sets used to generate presimulated data.
+            x: Simulation outputs of presimulated data.
+            from_round: Which round the data was simulated from. `from_round=0` means
+                that the data came from the first round.
```

Comment:
Will things work fine when providing data `from_round` > 0? If not, I'd suggest removing the keyword and documenting the assumptions on presimulated data in the docstring